backend/de.metas.business.rest-api-impl/src/main/java/de/metas/rest_api/invoicecandidates/impl/CheckInvoiceCandidatesStatusService.java

```diff
@@ -54,24 +55,18 @@
  * #L%
  */
 @Service
+@RequiredArgsConstructor
 public class CheckInvoiceCandidatesStatusService
 {
 	private final IInvoiceCandBL invoiceCandBL = Services.get(IInvoiceCandBL.class);
 	private final IInvoiceCandDAO invoiceCandDAO = Services.get(IInvoiceCandDAO.class);
 	private final IInvoiceDAO invoiceDAO = Services.get(IInvoiceDAO.class);
 
+	@NonNull
```

Comment:
pls keep the NonNull/Nullable annotations on the same line as the field?
(btw, do u have our coding style)